src/SprykerSdk/Integrator/IntegratorConfig.php

```diff
@@ -0,0 +1,289 @@
+    }
+
+    /**
+     * @return array<string>
+     */
+    public function getCoreNamespaces(): array
+    {
+        return $this->config[static::CORE_NAMESPACES];
+    }
+
+    /**
+     * @return array<string>
+     */
+    public function getCoreNonSplitOrganisations(): array
+    {
+        return [
+            'Spryker',
+            'SprykerShop',
+        ];
+    }
+
+    /**
+     * @return string
+     */
+    public function getProjectRootDirectory(): string
+    {
+        return APPLICATION_ROOT_DIR . DIRECTORY_SEPARATOR;
+    }
+
+    /**
+     * @return string
+     */
+    public function getCoreRootDirectory(): string
+    {
+        return APPLICATION_ROOT_DIR . DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR;
+    }
+
+    /**
+     * @return string
+     */
+    public function getNonSplitRepositoryPathPattern(): string
+    {
+        return 'spryker'
+            . DIRECTORY_SEPARATOR
+            . '%s'
+            . DIRECTORY_SEPARATOR
+            . 'Bundles'
+            . DIRECTORY_SEPARATOR;
+    }
+
+    /**
+     * @return string
+     */
+    public function getConfigVariableName(): string
+    {
+        return 'config';
+    }
+
+    /**
+     * @return string
+     */
+    public function getConfigPath(): string
+    {
+        return $this->getSharedConfigPath();
+    }
+
+    /**
+     * @return string
+     */
+    protected function getSharedConfigPath(): string
+    {
+        return $this->getProjectRootDirectory()
+            . 'config'
+            . DIRECTORY_SEPARATOR
+            . 'Shared'
+            . DIRECTORY_SEPARATOR
+            . 'config_default.php';
+    }
+
+    /**
+     * @return string
+     */
+    protected function getDefaultConfigPath(): string
+    {
+        return $this->getProjectRootDirectory() . 'config_integrator.php';
+    }
+
+    /**
+     * @return string
+     * @api
+     *
+     */
+    public function getIntegratorLockFilePath(): string
+    {
+        return $this->getProjectRootDirectory() . 'integrator.lock';
+    }
+
+    /**
+     * @return string
+     */
+    public function getComposerLockFilePath(): string
+    {
+        return $this->getProjectRootDirectory() . 'composer.lock';
+    }
+
+    /**
+     * @return string
+     */
+    public function getRecipesDirectory(): string
+    {
+        return $this->getProjectRootDirectory() . 'vendor/spryker-sdk/integrator/data/recipies/';
+    }
+
+    /**
+     * @return string
+     */
+    public function getRecipesRepository(): string
+    {
+        if (defined('TEST_INTEGRATOR_MODE')) {
+            return 'tests/_data/recipes/archive.zip';
+        }
+
+        return 'https://github.com/spryker-sdk/integrator-recipes/archive/master.zip';
+    }
+
+    /**
+     * @return array<string>
+     * @api
+     *
+     */
+    public function getInternalOrganizations(): array
+    {
+        return [
+            'Spryker',
+            'SprykerShop',
+            'SprykerMerchantPortal',
```

Comment:
Shouldn't we add it to getCoreNonSplitOrganisations method?

Also what is a difference between those 2 methods?